docs/guides/overview.md

```diff
@@ -0,0 +1,188 @@
+
+```js
+var Button = React.createClass({
+  render: function () {
+    return (
+      <button>
+        {this.props.children}
+      </button>
+    );
+  }
+});
+```
+
+Most of Radium's behavior is included in mixins that provide different types of functionality. For basic use, you'll want to start with the `StyleResolverMixin`. This mixin builds out the styles that you'll apply to your component.
+
+```js
+var Button = React.createClass({
+  mixins: [ StyleResolverMixin ],
+
+  render: function () { ... }
+});
+```
+
+`StyleResolverMixin` has a method called `buildStyles()` that converts a nested Radium style object into the styles that should be applied to your component based on its props and state. A basic Radium style object looks like this:
+
+```js
+{
+  standard: {
+    padding: '1.5em',
+    border: 0,
+    borderRadius: 4,
+    background: 'blue',
+    color: 'white',
+  }
+}
+```
+
+That object is passed as a parameter to `buildStyles()`. `buildStyles()` returns an object that can be applied to an element through the `style` attribute. If we assign our Radium style object to a variable called `radStyles`, that would look like this:
+
+```js
+var Button = React.createClass({
+  mixins: [ StyleResolverMixin ],
+
+  render: function () {
+    var styles = this.buildStyles(radStyles);
+
+    return (
+      <button style={styles}>
+        {this.props.children}
+      </button>
+    )
+  }
+});
+```
+
+From there, React will apply our `standard` styles to the `button` element. This is not very exciting. In fact, React does this by default, without the extra steps of nesting styles under `standard` and using `buildStyles()`. Radium becomes useful when you need to do more complex things, like handling modifiers, states, media queries, and computed properties.
+
+## Modifiers
+
+In Radium, a modifier is a set of additional CSS properties that are applied based on the component's props and state. A button might have modifiers to change its size or display property. Start by adding a `modifiers` property to your Radium style object.
+
+```js
+{
+  standard: { ... },
+  modifiers: {
+    size: {
+      large: {
+        fontSize: 24
+      },
+      small: {
+        fontSize: 12
+      }
+    },
+    block: {
+      display: 'block'
+    }
+  }
+}
+```
+
+Modifiers can reflect string or boolean values. If a modifier value is a string, you can represent different possible values as child objects with their own CSS properties:
+
+```js
+size: {
+  // if size === 'large'
+  large: {
+    fontSize: 24
+  },
+  // if size === 'small'
+  small: {
+    fontSize: 12
+  }
+}
+```
+
+If a modifier value is a boolean, add CSS properties as children of the modifier name:
+
+```js
+// if block === true
+block: {
+  display: 'block'
+}
+```
+
+By default, modifiers map directly to a component's `props`:
+
+```js
+<Button
```

Comment:
It might make sense to open the section with this example and then explain how it works.